calliope/backend/pyomo/constraints/energy_balance.py

```diff
@@ -211,7 +211,7 @@ def balance_supply_constraint_rule(backend_model, loc_tech, timestep):
     else:
         available_resource = resource * resource_scale
 
-    if po.value(force_resource):
+    if po.value(force_resource) == 1:
```

Comment:
Should we comment in cases like this that really this is boolean converted to 0/1? It is not very pythonic...

Reply:
It isn't, but is an issue with saving and loading the data to/from netcdf. In some instances, boolean data needs to be stored as floats, hence this (and some other) non-pythonic check...